apollo-openapi/src/main/java/com/ctrip/framework/apollo/openapi/dto/OpenCreateAppDTO.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2022 Apollo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.ctrip.framework.apollo.openapi.dto;
+
+import java.util.Set;
+
+public class OpenCreateAppDTO extends OpenAppDTO {
```

Comment:
Would it be better for OpenCreateAppDTO to composite OpenAppDTO rather than extending it?

Reply:
The json will be more complex to user?

from
```json
{
  "admins": [],
  "appId": "xxx",
  "ownerName": "yyy"
}
```

to

```json
{
  "admins": [],
  "app": {
      "appId": "xxx",
      "ownerName": "yyy"
  }

}
```

Reply:
Yes, but I suppose the user won't be aware of this? And it's kind of clear to me as the OpenAppDTO is separated from other properties of OpenCreateAppDTO, such as `admins` and `assignAppRoleToSelf`.

Reply:
Done.